web/src/components/dateRangePicker/DateRangePickerWrapper.tsx

```diff
@@ -30,14 +31,14 @@ const DateRangePickerWrapper: React.FunctionComponent<DateRangePickerWrapperProp
   const handleKeyPress = (event: any) => event?.key === 'Escape' && handleToggle()
 
   const minDate = initialDateRange.startDate || new Date()
-  const maxDate = DateTime.fromJSDate(minDate).plus({ days: 6 }).toJSDate()
+  const maxDate = DateTime.fromJSDate(minDate).plus({ days: 11 }).toJSDate()
```

Comment:
Should technically be plus 10 days. The 11 days specified in the ticket counts Day 1 as today.

Reply:
The date range is inclusive so this is already the case

Reply:
![Screenshot 2023-03-09 at 12 16 25 PM](https://user-images.githubusercontent.com/54914403/224145149-f833282b-4dd0-4a96-ac41-9b3dbb28a307.png)

Load page -> open datepicker without changing the dates selected - the 12th day is clickable when it shouldn't be. But really I'm not worried about this one so much, but the extra day showing up repeatedly in the table with N/A every time needs to be fixed

Reply:
Again, not able to reproduce...

Reply:
Are you still able to reproduce the extra-day-in-the-table bug @dgboss ?

Reply:
Day offset fixed

Reply:
Yes, I can reproduce. I added a comment describing where it can be fixed (initialization of the endDate).